nilearn/decoding/searchlight.py

```diff
@@ -328,6 +330,25 @@ def __init__(
         self.cv = cv
         self.verbose = verbose
 
+    def _more_tags(self):
+        return self.__sklearn_tags__()
+
+    def __sklearn_tags__(self):
+        # TODO
+        # get rid of if block
+        # bumping sklearn_version > 1.5
+        ver = parse(sklearn_version)
+        if ver.release[1] < 6:
+            from nilearn._utils.tags import tags
+
+            return tags()
+
+        from nilearn._utils.tags import InputTags
+
+        tags = super().__sklearn_tags__()
+        tags.input_tags = InputTags()
+        return tags
```

Comment:
I have had to add this pattern to almost all our estimators.
it's a bit ugly but it gets the job done